### PR TITLE
fix(openapi): add base url

### PR DESCRIPTION
**Pull request by BlairCurrey**
## Changes proposed in this pull request
- adds `rafiki.money` as base path

image

## Context
- fixes https://github.com/interledger/open-payments/issues/303

The issues suggests maybe using rafiki.money as the base url but I opted for openpayments.guide because that's what it originally was. https://github.com/interledger/open-payments/blob/5d39b2ef5f202e6378555b790730f9465ecd6f97/openapi/resource-server.yaml#L35C3-L39 This makes the base url consistent with existing examples.

### CODE REVIEW

File: openapi/resource-server.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ paths:
                 New Incoming Payment for $25:
                   value:
                     id: 'https://openpayments.guide/incoming-payments/08394f02-7b7b-45e2-b645-51d04e7c330c'
-                    walletAddress: 'https://openpayments.guide/alice/'
```
Comment (njlie) [inline openapi/resource-server.yaml:115]: I believe `walletAddress` values should still have `/alice`. That seems like the case to me but I could be wrong @mkurapov / @sabineschaller

File: .changeset/loud-snakes-melt.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': major
```
Comment (njlie) [inline .changeset/loud-snakes-melt.md:2]: I don't think this constitutes a major change. In fact, I'm not sure this changes anything in the `open-payments` package itself at all.

File: openapi/resource-server.yaml [hunk 3]
```diff
@@ -32,8 +32,8 @@ info:
   contact:
     email: tech@interledger.org
 servers:
-  - url: '/'
-    description: 'Server for wallet address subresources or Connection resources (ie https://openpayments.guide/alice)'
+  - url: 'https://openpayments/'
```
Comment (mkurapov) [inline openapi/resource-server.yaml:35]: Not sure if I have a preference, but `rafiki.money` might be nice to have if we are going to include some code snippets in the documentation that _do_ use `rafiki.money`. I would connect with Radu /Melissa/Mohammed for this

File: openapi/resource-server.yaml (hunk 1 above)
Comment (mkurapov) [reply openapi/resource-server.yaml:115]: Yeah, the wallet address remains the same, just the baseUrl changes

File: .changeset/loud-snakes-melt.md (hunk 2 above)
Comment (BlairCurrey) [reply .changeset/loud-snakes-melt.md]: removed

File: openapi/resource-server.yaml (hunk 3 above)
Comment (BlairCurrey) [reply openapi/resource-server.yaml]: I will change to rafiki.money. Melissa confirmed that there will be code snippets in the docs that are intended to be used with rafiki.money. https://github.com/interledger/open-payments-snippets She also noted the openpayments.guide domain might change as well but in any case it seems like the rafiki.money url will be more useful.

File: openapi/resource-server.yaml (hunk 1 above)
Comment (BlairCurrey) [reply openapi/resource-server.yaml:115]: this and similar are now `https://rafiki.money/alice/`

**Review by mkurapov — APPROVED** [review]

**Review by njlie — APPROVED** [review]

**Pull request merged**